examples/jira_mysql_install.pp

```diff
@@ -0,0 +1,24 @@
+node default {
+
+  class { '::mysql::server':
+    root_password    => 'strongpassword',
+  } ->
+
+  mysql::db { 'jira':
+    user     => 'jiraadm',
+    password => 'mypassword',
+    host     => 'localhost',
+    grant    => ['ALL'],
+  } ->
+
+  class { '::jira':
+    javahome    => '/opt/java/latest',
+    db          => 'mysql',
+    dbport      => '3306',
+    dbdriver    => 'com.mysql.jdbc.Driver',
+    dbtype      => 'mysql',
+  } ->
+
+  class { '::jira::facts': }
```

Comment:
I can't say [no](https://github.com/ffrank/puppetlint-classdeclerations) to you guys.